lang/src/org/partiql/lang/ast/passes/AstWalker.kt

```diff
@@ -16,6 +16,7 @@ package org.partiql.lang.ast.passes
 
 import org.partiql.lang.ast.*
 import org.partiql.lang.util.*
+import java.lang.UnsupportedOperationException
```

Comment:
is this import needed? looks like a left-over

Reply:
Yes, that is leftover. Removed in the new commit. Thanks!